docs/source/user-guide/getting-started.rst

```diff
@@ -43,7 +43,7 @@ Conda is available on Windows, macOS, or Linux and can be used with any terminal
 
    .. tab-item:: Windows
 
-      #. Open either the Command Prompt (cmd.exe) or PowerShell.
+      #. Open either the Anaconda Command Prompt (cmd.exe) or Anaconda PowerShell.
```

Comment:
```suggestion
      #. Open either the Anaconda Command Prompt (cmd.exe) or Miniforge Command Prompt (Powershell prompts are available too).
```

@jezdez, how do you like the wording on that?

Reply:
```suggestion
      #. Open either the Anaconda or Miniforge Command Prompt (cmd.exe), depending on your installation. Powershell prompts are also available.
```
A simplification. Also, not sure if "installation" or "distribution" is the better word here. Does Miniforge also have a Powershell prompt available? That's what the second sentence is currently implying.

Reply:
I'm not sure 😅. Maybe we could change it to, "Powershell propmts may also be available"?

Reply:
Are just leave Powershell out entirely.

Reply:
I installed Miniforge on my Windows VM and I'm only finding Miniforge Prompt. No Powershell that I can see. We could say, "A PowerShell prompt is also available with Anaconda Distribution or Miniconda." Or something like that?